Src/Plugins/Tests.SalModernization/TestData/UpdateSalToCurrentVersion/ExpectedOutputs/SEC105_001.AnalysisAssume.sarif

```diff
@@ -14,80 +14,48 @@
+                "text": "SAL 1 can be replaced with SAL 2."
               },
               "messageStrings": {
                 "NotApplicable_InvalidMetadata": {
                   "text": "'{0}' was not evaluated for check '{1}' because the analysis is not relevant for the following reason: {2}."
                 },
                 "Default": {
-                  "text": "'{0}' contains incorrect or out-dated use of SAL."
+                  "text": "'{0}' contains a call to the SAL v1 '{1}' annotation and should be replaced for '{2}'."
                 }
               },
               "helpUri": "https://github.com/microsoft/sarif-pattern-matcher"
             },
             {
-              "id": "SEC105/119",
-              "name": "UpdateSalToCurrentVersion/Maybenull",
+              "id": "SEC105/003",
+              "name": "ConvertObsoleteAnnotation",
               "fullDescription": {
-                "text": "Update SAL to current version"
+                "text": "Conversion from SAL 1 to SAL 2 cannot be automatically done."
               },
               "messageStrings": {
                 "NotApplicable_InvalidMetadata": {
                   "text": "'{0}' was not evaluated for check '{1}' because the analysis is not relevant for the following reason: {2}."
                 },
                 "Default": {
-                  "text": "'{0}' contains incorrect or out-dated use of SAL."
-                }
-              },
-              "helpUri": "https://github.com/microsoft/sarif-pattern-matcher"
-            },
-            {
-              "id": "SEC105/121",
-              "name": "UpdateSalToCurrentVersion/Notnull",
-              "fullDescription": {
-                "text": "Update SAL to current version"
-              },
-              "messageStrings": {
-                "NotApplicable_InvalidMetadata": {
-                  "text": "'{0}' was not evaluated for check '{1}' because the analysis is not relevant for the following reason: {2}."
-                },
-                "Default": {
-                  "text": "'{0}' contains incorrect or out-dated use of SAL."
-                }
-              },
-              "helpUri": "https://github.com/microsoft/sarif-pattern-matcher"
-            },
-            {
-              "id": "SEC105/159",
-              "name": "UpdateSalToCurrentVersion/PostInvalid",
-              "fullDescription": {
-                "text": "Update SAL to current version"
-              },
-              "messageStrings": {
-                "NotApplicable_InvalidMetadata": {
-                  "text": "'{0}' was not evaluated for check '{1}' because the analysis is not relevant for the following reason: {2}."
-                },
-                "Default": {
-                  "text": "'{0}' contains incorrect or out-dated use of SAL."
+                  "text": "'{0}' contains a call to the SAL v1 '{1}' annotation and should be converted manually."
```

Comment:
>call [](http://example.com/codeflow?start=44&length=4)

There are no 'calls' to annotations, annotations are applied to code. i.e., they are metadata only, not executable.

'{0}' contains a use of the SAL v1 '{1} annotation that should be converted manually to the correct SAL v2 pattern.

Reply:
fixed.